src/zzz_od/application/hollow_zero/lost_void/context/lost_void_context.py

```
@@ -717,26 +717,37 @@ def fmt_artifact(pos: LostVoidArtifactPos, idx: int | None = None) -> str:
-        priority_list_to_consider = []
-
-        final_priority_list_1 = self.dynamic_priority_list.copy()
-        if consider_priority_1 and self.challenge_config.artifact_priority_in_battle:
-            final_priority_list_1.extend(self.challenge_config.artifact_priority_in_battle)
-        priority_list_to_consider.append(final_priority_list_1)
-
-        if consider_priority_2 and self.challenge_config.artifact_priority_2:
-            priority_list_to_consider.append(self.challenge_config.artifact_priority_2)
-
-        if len(priority_list_to_consider) == 0:  # 两个优先级都是空的时候 强制考虑非优先级的
+        # 六级优先级模型，按顺序匹配：
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)